common/version.go

```diff
@@ -17,4 +17,4 @@
 package common
 
 // Version of the SDK
-const Version = "2.0.5"
+const Version = "2.0.6"
```

Comment:
This change should be removed. Publication automation bumps version automatically.